### ISSUE TITLE
Quaver does not set pause timeout when the user leaves right after Quaver joins a channel.

### ISSUE THREAD

**Issue by Xayanide** [ZPTXDev/Quaver#174]
**Describe the bug**
What isn't working as intended, and what does it affect?
Pause time out feature
Affects user experience
Is not reproducible with low latency

**Affected versions**
What versions are affected by this bug? (e.g. >=3.0.1, 2.5.1-2.6.3, >=1.2.0)

This was probably supposed to be found earlier, I'd say roughly as early as around 
2.4.0

**Steps to reproduce**
Steps to reproduce the behavior. (e.g. click on a button, enter a value, etc. and see error)
1. Join either voice or stage channel
2. Play any track with Quaver
3. As Quaver joins the voice channel you're in, immediately leave the channel.
4. Must be done in a precise manner.

**Search**
1. Join either voice or stage channel
2. Search any track
3. Pick one or more tracks, do not click away yet.
4. Move your cursor to the disconnect button.
5. Click away to add tracks to queue
6. Disconnect from the channel
7. Must be done in a precise manner.

**Expected behavior**
What is expected to happen?
Set pause timeout when no humans or users are in the channel while Quaver is playing

**Actual behavior**
What actually happens? Attach or add errors or screenshots here as well.
Quaver did not set pause timeout, continues playing the track.

Console log of the entire reproduction process.

```js
2022-04-04T17:11:50.611Z [Discord] INFO: Connected. Logged in as ThatKid#6969.
2022-04-04T17:11:50.614Z [Quaver] INFO: Running version 3.4.0-next.39. For help, see https://github.com/ZapSquared/Quaver/issues.
2022-04-04T17:11:50.615Z [Quaver] WARN: You are running an unstable version of Quaver. Please report bugs using the link above, and note that f
eatures may change or be removed entirely prior to release.
2022-04-04T17:11:51.321Z [Lavalink] INFO: Connected.
2022-04-04T17:12:04.212Z [Quaver] INFO: [G 906471497231630336 | U 250934589621665793] Processing command play
2022-04-04T17:12:04.215Z [Quaver] INFO: [G 906471497231630336 | U 250934589621665793] Executing command play
// user is no longer in the channel at this time, Quaver did not set a pausetimeout
```

# Update as of  3.4.2-next.3

## Play command

What I did in the video demo:
- Do the reproduction steps
- Wait for the track to end

https://user-images.githubusercontent.com/18362871/164445720-a8c2ab61-56c6-4099-9e64-dccca13a3173.mp4

Full console log
```js
$ node .
2022-04-21T11:07:46.328Z [Discord] INFO: Connected. Logged in as Aia#3990.
2022-04-21T11:07:46.335Z [Quaver] INFO: Running version 3.4.2-next.3. For help,
see https://github.com/ZapSquared/Quaver/issues.
2022-04-21T11:07:46.335Z [Quaver] WARN: You are running an unstable version of Q
uaver. Please report bugs using the link above, and note that features may chang
e or be removed entirely prior to release.
2022-04-21T11:07:47.164Z [Lavalink] INFO: Connected.
2022-04-21T11:07:51.630Z [Quaver] INFO: [G 906471497231630336 | U 25093458962166
5793] Processing command play
2022-04-21T11:07:51.634Z [Quaver] INFO: [G 906471497231630336 | U 25093458962166
5793] Executing command play
2022-04-21T11:07:54.197Z [Quaver] INFO: [G 906471497231630336] Starting track
2022-04-21T11:08:10.524Z [Quaver] INFO: [G 906471497231630336] Disconnecting (al
one)
2022-04-21T11:08:10.528Z [Quaver] INFO: [G 906471497231630336] Setting timeout
```

## Search command

What I did in the video demo:
- Do the reproduction steps
- Wait for the track to end

https://user-images.githubusercontent.com/18362871/164444697-d122fb02-8ca8-40f2-8f7c-fb6f0ada7e4a.mp4

Full console log

```js
$ node .
2022-04-21T10:59:58.584Z [Discord] INFO: Connected. Logged in as Aia#3990.
2022-04-21T10:59:58.588Z [Quaver] INFO: Running version 3.4.2-next.3. For help,
see https://github.com/ZapSquared/Quaver/issues.
2022-04-21T10:59:58.589Z [Quaver] WARN: You are running an unstable version of Q
uaver. Please report bugs using the link above, and note that features may chang
e or be removed entirely prior to release.
2022-04-21T10:59:59.388Z [Lavalink] INFO: Connected.
2022-04-21T11:00:04.471Z [Quaver] INFO: [G 906471497231630336 | U 25093458962166
5793] Processing command search
2022-04-21T11:00:04.475Z [Quaver] INFO: [G 906471497231630336 | U 25093458962166
5793] Executing command search
2022-04-21T11:00:12.338Z [Quaver] INFO: [G 906471497231630336] Starting track
2022-04-21T11:00:28.501Z [Quaver] INFO: [G 906471497231630336] Disconnecting (al
one)
2022-04-21T11:00:28.504Z [Quaver] INFO: [G 906471497231630336] Setting timeout
```

**Comment by Xayanide** [comment]
The absurdity and rarity of this condition will make this issue invalid and will be closed some time soon.

**Comment by zapteryx** [comment]
doubt i'll close it, but maybe it won't be patched by 3.4.0 and will instead be delayed to 3.4.1

**Comment by Xayanide** [comment]
Updated.
